README.md

```diff
@@ -202,6 +202,7 @@ If `ruby-kafka` doesn't fit your kafka environment, check `rdkafka2` plugin inst
       headers_from_record   (hash)   :default => {}
       use_default_for_unknown_topic (bool) :default => false
       discard_kafka_delivery_failed (bool) :default => false (No discard)
+      partitioner_hash_function (enum) (crc32|murmur32) :default => 'crc32'
 
```

Comment:
Basically, giving a choice who can select hash algorithm is ok, but the default hash function may be 
better to change (from crc32 to murmur32 IMHO)

It breaks a compatibility, yes, so it should bump the version and add a note why the default partitioner hash
algorigthm is changed - it should follow the behavior of  kafka-console-producer for consistency.